packages/p2p-v2/src/pages/orders/components/OrderDetailsCard/OrderDetailsCardFooter/__tests__/OrderDetailsCardFooter.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useOrderDetails } from '@/pages/orders/screens/OrderDetails/OrderDetailsProvider';
+import { render, screen } from '@testing-library/react';
+import OrderDetailsCardFooter from '../OrderDetailsCardFooter';
+
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: () => ({ isMobile: false }),
+}));
+
+jest.mock('@/pages/orders/screens/OrderDetails/OrderDetailsProvider', () => ({
+    useOrderDetails: jest.fn().mockReturnValue({
+        shouldShowCancelAndPaidButton: true,
+        shouldShowComplainAndReceivedButton: false,
+        shouldShowOnlyComplainButton: false,
+        shouldShowOnlyReceivedButton: false,
```

Comment:
@ameerul-deriv, wouldn't it be better to put these in an `object`, spread it, then overwrite the ones you need to change? 🤔

Reply:
true can do that 👍